code/controllers/subsystem/deputy.dm

```diff
@@ -0,0 +1,67 @@
+SUBSYSTEM_DEF(deputy)
+	name = "Deputy"
+	offline_implications = "Deputy status of crew will no longer automatically update. No immediate action is needed."
+	runlevels = RUNLEVEL_GAME
+	wait = 1 MINUTES
+
+	var/last_deputized = FALSE
+
+/datum/controller/subsystem/deputy/Initialize()
+	// if we aren't using the deputy subsystem, don't fire it
+	if(!config.use_deputy_subsystem)
+		flags |= SS_NO_FIRE
+
+	// call parent initialization routine
+	return ..()
+
+/datum/controller/subsystem/deputy/fire()
+	// first determine how many active security people we have on station
+	var/list/active_with_role = number_active_with_role()
+	var/num_security = active_with_role["Security"]
+
+	// next determine how many active players we have
+	var/num_players = active_with_role["Any"]
+
+	// if there are no players, there is nobody to deputize
+	if(num_players < 1)
```

Comment:
if(!num_players)

Reply:
I'll just leave this as-is. I'm not a fan of implicit casts from integer to boolean.